XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseController.cs

```diff
@@ -246,7 +246,7 @@ internal async Task TryRenderControllerModelAsync(Type controllerType, byte[] gl
                 if (useSystemDefaultModels && gltfObject != null)
                 {
                     controllerModel.name = $"{controllerType.Name}_Visualization";
-                    controllerModel.transform.SetParent(MixedRealityToolkit.CameraSystem?.CameraRig.PlayspaceTransform);
+                    controllerModel.transform.SetParent(MixedRealityToolkit.CameraSystem?.MainCameraRig.PlayspaceTransform);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main`